### PR TITLE
[zh-CN]: sync Notification.permission static property with en-US

### PR DESCRIPTION
**Pull request by skyclouds2001**
### Description

sync Notification.permission static property with en-US

### Motivation

sync Notification.permission static property with en-US

### Additional details

[https://developer.mozilla.org/en-US/docs/Web/API/Notification/permission_static](https://developer.mozilla.org/en-US/docs/Web/API/Notification/permission_static)
[https://developer.mozilla.org/zh-CN/docs/Web/API/Notification/permission_static](https://developer.mozilla.org/zh-CN/docs/Web/API/Notification/permission_static)

### CODE REVIEW

File: files/zh-cn/web/api/notification/permission_static/index.md [hunk 1]
```diff
@@ -1,70 +1,61 @@
...
+title: Notification：permission 静态属性
 slug: Web/API/Notification/permission_static
-original_slug: Web/API/Notification/permission
 ---
 
-{{APIRef("Web Notifications")}}
+{{APIRef("Web Notifications")}}{{AvailableInWorkers}}{{securecontext_header}}
 
-`Notification` 的只读属性 permission 用来表明用户是否允许当前域显示 Web Notification。
+{{domxref("Notification")}} 的 `permission` 只读属性表明当前用户是否授予当前来源显示系统通知的权限。
```
Comment (jasonren0403) [inline files/zh-cn/web/api/notification/permission_static/index.md:8]: https://developer.mozilla.org/zh-CN/docs/Glossary/Origin

```suggestion
{{domxref("Notification")}} 接口的只读属性 `permission` 表明当前用户是否授予当前源显示 web 通知的权限。
```

File: files/zh-cn/web/api/notification/permission_static/index.md [hunk 2]
```diff
@@ -1,70 +1,61 @@
...
-
-### Value
-
-permission 的类型为 {{domxref("DOMString")}} . 该属性的可能值为：
+- `granted`
+  - : 用户已经明确地授予了当前来源显示系统通知的权限。
+- `denied`
+  - : 用户已经明确地拒绝了当前来源显示系统通知的权限。
+- `default`
+  - : 用户是否授予当前来源显示系统通知的权限的决定是未知的；在这种情况下，应用将表现得好像权限被“拒绝”一样
```
Comment (jasonren0403) [inline files/zh-cn/web/api/notification/permission_static/index.md:19]:
```suggestion
  - : 用户是否授予当前来源显示系统通知的权限的决定是未知的；在这种情况下，应用的行为与该值为 `denied` 的情况相同。
```

**Review by jasonren0403 — COMMENTED** [review]
some suggestions

File: files/zh-cn/web/api/notification/permission_static/index.md [hunk 3]
```diff
@@ -1,70 +1,61 @@
...
+title: Notification：permission 静态属性
 slug: Web/API/Notification/permission_static
-original_slug: Web/API/Notification/permission
 ---
 
-{{APIRef("Web Notifications")}}
+{{APIRef("Web Notifications")}}{{AvailableInWorkers}}{{securecontext_header}}
 
-`Notification` 的只读属性 permission 用来表明用户是否允许当前域显示 Web Notification。
+{{domxref("Notification")}} 接口的只读属性 `permission` 表明当前用户是否授予当前源显示 Web 通知的权限。
```
Comment (yin1999) [inline files/zh-cn/web/api/notification/permission_static/index.md:8]:
```suggestion
{{domxref("Notification")}} 接口的只读属性 `permission` 表明当前用户是否授予当前来源（origin）显示 web 通知的权限。
```

File: files/zh-cn/web/api/notification/permission_static/index.md [hunk 4]
```diff
@@ -1,70 +1,61 @@
...
 
-{{AvailableInWorkers}}
+## 值
 
-## Syntax
+一个表示当前权限的字符串，值可以是：
```
Comment (yin1999) [inline files/zh-cn/web/api/notification/permission_static/index.md:12]:
```suggestion
一个表示当前权限的字符串。其值可以是：
```

File: files/zh-cn/web/api/notification/permission_static/index.md [hunk 5]
```diff
@@ -1,70 +1,61 @@
...
 
-## See also
+## 参见
 
-- [Using the Notifications API](/zh-CN/docs/Web/API/Notifications_API/Using_the_Notifications_API)
-- {{domxref("Permissions_API","Permissions API")}}
+- [Notifications API](/zh-CN/docs/Web/API/Notifications_API)
+- [使用 Notifications API](/zh-CN/docs/Web/API/Notifications_API/Using_the_Notifications_API)
+- [Permissions API](/zh-CN/docs/Web/API/Permissions_API)
+- [使用 Permissions API](/zh-CN/docs/Web/API/Permissions_API/Using_the_Permissions_API)
```
Comment (yin1999) [inline files/zh-cn/web/api/notification/permission_static/index.md:61]:
```suggestion
- [Notification API](/zh-CN/docs/Web/API/Notifications_API)
- [使用 Notification API](/zh-CN/docs/Web/API/Notifications_API/Using_the_Notifications_API)
- [Permission API](/zh-CN/docs/Web/API/Permissions_API)
- [使用 Permission API](/zh-CN/docs/Web/API/Permissions_API/Using_the_Permissions_API)
```

**Review by yin1999 — APPROVED** [review]
Thanks

**Pull request merged**